File: feitos/distanciapontos.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main(){
	double x1, x2, x, y1, y2, y, dis;
	
	cin >> x1 >> y1;
	cin >> x2 >> y2;
	
	std::cout.precision(4);
	std::cout.setf(std::ios::fixed, std::ios::floatfield);
	
	x = pow((x2 - x1), 2);
	y = pow((y2 - y1), 2);
	
	dis = sqrt(x + y);	
	
	cout << dis << endl;
	return 0;
}

File: feitos/distanciapontos.py
from math import sqrt

dado1 = input()
dado2 = input()

x1, y1 = dado1.split(" ")
x2, y2 = dado2.split(" ")

x = pow((float(x2) - float(x1)), 2)
y = pow((float(y2) - float(y1)), 2)

dis = sqrt(x + y)

print("%.4f" % dis)